### ISSUE TITLE
Blank OreRegistry.json

### ISSUE THREAD

**Issue by m1jordanallen** [MikeLydeamore/ExNihiloAdscensio#133]
It seems that as of version 0.1.18 to current, The OreRegistry.json comes completely blank aside from a couple brackets.  I checked version 0.1.17 and that seems to load just properly.  

I will be using the older version to register the json as a workaround, but figured I would pass this finding along.  

Thank you for the mod!

**Comment by BrianEdmonds** [comment]
I noticed this in v19 as well, but couldn't confirm if it was a problem
on its own, or if it was a side effect of the stack overflow crash. I've
not created a new config with v20 yet.

**Comment by m1jordanallen** [comment]
To cut out all other variables I used current version of forge, and the old recommended build, and limited it to just forge and ex nihilo adscensio.  If that helps locate the issue at all :)

*edit(added info):v20 had the issue as well as stated "to current".

**Comment by darthvader45** [comment]
Agreed, same problem.  Gonna try .17 and see what happens.

**Comment by darthvader45** [comment]
Okay, workaround works.  Dunno how to sift to get the pieces.  JEI and NEI don't show any recipes for them.

**Comment by mcgay** [comment]
Figured out a solution, it's permanent (i think?) but requires some repetitive file changing

Short version : Get v17 and latest version, get the v17 text files for OreRegistry.json and SieveRegistry.json, then copy that in the latest version.

Explain to me like I'm 5 :
>Download both v17 and the version you want (latest is recommended!)
>Put the v17 in the mods folder, and if you had a more later version then delete the config for that file
  >Delete the "exnihiloadscensio" folder in the "config" file in ".minecraft"
>Launch minecraft and play in a world
>Now the exnihilo config folder has been created
>Create a new folder in your desktop (
>Open the exnihiloadscensio folder in the config folder
>Open both "OreRegistry.json" and "SieveRegistry.json" as Notepad
>Create new text files in the folder you created on the desktop (for less confusion, name one OreRegistry.json , and the other one "SieveRegistry.json"
>Copy whatever is in the OreRegistry.json to the text file, do the same with SieveRegistry.json
>Now you should have text files in the folder that you created on desktop that have the text from v17 OreRegistry.json and SieveRegistry.json
>Delete the "exnihiloadscensio" folder in the "config" file, and do the same with the mod
>Put the latest version in the mods file
>Launch minecraft and play in a world, which should create the new config folder
>Find OreRegistry.json and SieveRegistry.json in the new "exnihiloadscensio" folder
>Remove everything from them, and copy in the text that you have in your text files which is in your new folder on the desktop
>OreRegistry.json text which is in your folder on desktop needs to be copied in the main text file that is in your config folder, same with SieveRegistry.json
>Save it, and run minecraft, and it will all work 👍

**Comment by darthvader45** [comment]
Yeah, though if it could be added by default, it'd be better.  Either way, I have Creatio now, so it should be a bit easier now, since that bug is likely fixed already.
